ScarletEngine/Sources/Gameplay/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ set(Gameplay_PrivateDeps
 set(Gameplay_PublicDeps
 	"ECS"
 	"Engine"
-	"RenderModule")
+	"RenderModule"
+	"AchillesPhysics")
```

Comment:
Why does Gameplay depend on Achilles?
Did you mean to do the inverse and make Achilles depend on Gameplay? If so you need to make that change in the CMakeLists.txt inside of Achilles, not gameplay.